modules/network-observability-flowcollector-view.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-flowcollector-view_{context}"]
+
+= View the FlowCollector resource 
+ You can view and edit YAML directly in the {product-title} web console. In the web console, navigate to *Operators* -> *Installed Operators*. Then under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. Next, select *cluster* then select the *YAML* tab. There, you can modify the `FlowCollector` resource to configure the Network Observability operator. Some of the notable configurations are documented in this documentation. 
```

Comment:
This text is not rendered correctly, maybe due to the leading space before "You can ..." ?

![image](https://user-images.githubusercontent.com/2153442/208851741-8bdb3ff8-7f76-4ed1-9ecc-4f0b93fde2f5.png)

Reply:
Ugh, yep need to backspace here